docs/developer-docs/latest/getting-started/introduction.md

```diff
@@ -6,38 +6,38 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/getting-started/intro
 
 # Welcome to the Strapi v4 developer documentation!
 
-This documentation contains all technical documentation related to the setup, deployment, update and customization of your Strapi application.
+This documentation contains all technical documentation related to the setup, deployment, update and customization of a Strapi application.
 
 ::: strapi Can't wait to start using Strapi?
 You can directly head to the [Quick Start](quick-start.md)! <br> If demos are more your thing, we have a [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo)!
 :::
 
-The original purpose of the project was to help Boot**strap** your **API**: that's how Strapi was created. Now, Strapi is an open-source headless CMS that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel. Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible - and which every part is customizable to match any use case. Strapi also has a built-in user system to manage in detail what the administrators and end users have access to.
+The original purpose of the project was to help Boot**strap** an **API**: that's how Strapi was created. Now, Strapi is an open-source headless CMS that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel. Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible – and every part of which is customizable to match any use case. Strapi also has a built-in user system that allows to create fine-grained access controls for administrators as well as end users.
```

Comment:
This isn't true of end users, the admin RBAC system is more complex and offers more control then the current end users system.

This new sentence will cause more confusion then the current one does.

Reply:
OK, this is a case where I do not have enough technical insight. I would nevertheless vote for a nicer formulation...

Reply:
```suggestion
The original purpose of the project was to help Boot**strap** your **API**: that's how Strapi was created. Now, Strapi is an open-source headless CMS that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel. Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible - and which every part is customizable to match any use case. Strapi also has a built-in user system to manage, in detail, what the administrators and end users have access to.
```

This probably needs more work once we finally stop installing the users-permissions plugin by default. for now I just added a sentence break but outside of that I don't think it needs anything else.